integrasjonstest/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/integrasjonstest/InnsendingIT.kt

```diff
@@ -237,9 +239,17 @@ class InnsendingIT : EndToEndTest() {
         const val JOURNALPOST_ID = "journalpost-id-skoleboller"
         const val SAK_ID = "forundret-lysekrone"
         const val OPPGAVE_ID = "neglisjert-sommer"
+        val orgnr = Orgnr.genererGyldig()
 
         val forespoerselId: UUID = UUID.randomUUID()
-        val skjema = mockInnsending().copy(identitetsnummer = bjarneBetjent.ident!!)
+        val skjema =
+            mockInnsending().let { innsending ->
+                innsending.copy(
+                    identitetsnummer = bjarneBetjent.ident!!,
+                    orgnrUnderenhet = orgnr.toString(),
+                    bestemmendeFraværsdag = innsending.fraværsperioder.lastOrNull()?.fom ?: innsending.bestemmendeFraværsdag,
```

Comment:
Hva skyldes denne endringen? Alt som gjør noe med bestemmede fraværsdag skremmer meg 🫣

Reply:
Godt spørsmål, for det husker jeg ikke lenger 😅 Uansett så har jeg byttet det ut med `mockSkjemaInntektsmelding()`-metoden som du lagde i `MockInntektsmelding.kt`, så dette er borte nå 🗑️